fujiji-client/src/components/Listing/Listing.jsx

```diff
@@ -52,47 +64,114 @@ export default function Listing({
   onContactClick = () => {},
   contactEmail,
 }) {
+  const router = useRouter();
+  const toast = useToast();
+  const { result, packageID } = router.query;
+  const { authToken } = useSession();
+
+  useEffect(() => {
```

Comment:
Is this `useEffect` does not have any dependencies? This might cause the infinite re-render thing like before

Reply:
It is not causing any issues so far